lib/modules/neverEndingReddit.js

```diff
@@ -167,7 +167,12 @@ function togglePause(pause, source) {
 }
 
 function initiateReturnToPrevPage() {
+	const navigation = window.performance.navigation;
```

Comment:
Nit: use `performance.navigation` for future type checking (`window` is untyped)